Presentation/Presentation/Sources/Wordle/ViewModel/WordleViewModel.swift

```diff
@@ -146,25 +171,36 @@ final class WordleViewModel: ObservableObject {
         }
         triedWordleCount += 1
 
-        if gameObjcet.gameAnswer == submitWordle.joined() || triedWordleCount == 6 {
-            isGameOver = true
-            canSubmitWordle = false
+        guard gameObject.gameAnswer == submitWordle.joined() || triedWordleCount == 6  else { return }
+        isGameOver = true
+        canSubmitWordle = false
+    }
+
+    private func loadWordleHistory() {
+        let wordleHistory = gameRepository.loadWordleHistory(gameID: gameObject.id)
+        for word in wordleHistory {
+            let wordArray = word.map { String($0) }
+            for index in 0..<wordleWordCount {
+                wordle[triedWordleCount][index].alphabet = wordArray[index]
+            }
+            changeWordleState(submitWordle: wordArray)
         }
     }
 
-    private func changeKeyboardState(alphabet: String, state: KeyboardState) {
-        for index in 0..<3 {
-            guard let keyboardIndex = keyboard[index].firstIndex(where: { $0.alphabet == alphabet}) else { continue }
-            switch keyboard[index][keyboardIndex].keyboardState {
-            case .correct:
+    private func saveWordleHistory() {
+        var wordleHistory: [String] = []
+        for word in wordle {
+            guard word.filter({ $0.alphabet != nil }).count == 5,
+                  !word.contains(where: { $0.state == .typing })
+            else {
```

Comment:
단순 스타일입니다만, 저희 컨벤션은 아래가 맞는 것 같습니다!
```suggestion
            guard
                 word.filter({ $0.alphabet != nil }).count == 5,
                 !word.contains(where: { $0.state == .typing })
            else {
```

Reply:
굿뽀이 ~ 바꿨습니다 !!